plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/QUtils.kt

```diff
@@ -14,6 +14,8 @@ import software.aws.toolkits.jetbrains.core.credentials.pinning.CodeWhispererCon
 import software.aws.toolkits.jetbrains.core.credentials.sono.isSono
 import software.aws.toolkits.jetbrains.services.telemetry.ClientMetadata
 
+fun isInternalUser(startUrl: String?): Boolean = startUrl == "https://amzn.awsapps.com/start"
```

Comment:
This function can probably be ported over to Core since it can be used for both plugins

Reply:
ok

Reply:
moved to https://github.com/aws/aws-toolkit-jetbrains/pull/5019/files#diff-12f7a9784177d4db55b719d0a41b57fc140b58be2e21d8805e5b207ce4237d6cR38